docs/guides/SimpleStartupGuide.md

```diff
@@ -0,0 +1,414 @@
+# USB Camera Device Service Simple Startup Guide
+
+## Contents
+
+[Overview](#overview)  
+[System Requirements](#system-requirements)  
+[How It Works](#how-it-works)  
+[Tested Devices](#tested-devices)  
+[Dependencies](#dependencies)  
+[Get the Source Code](#get-the-source-code)  
+[Deploy the Service](#deploy-edgex-and-usb-device-camera-microservice)  
```

Comment:
This section is also not used. However, the "Run the Service" tab is included under "get the source code," I think it might be best to break it up into its own section